opba-db/src/main/resources/migration/migrations/0002-mock-data.xml

```diff
@@ -148,4 +148,74 @@
             </where>
         </update>
     </changeSet>
+
+    <changeSet author="iiuex@adorsys.com.ua" id="2020-08-21-1">
+        <loadData
+                encoding="utf8"
+                relativeToChangelogFile="true"
+                file="bank_profile_data.csv"
+                catalogName=""
+                tableName="${table-prefix}bank">
+            <column header="uuid"       name="uuid"         type="STRING"/>
+            <column header="name"       name="name"         type="STRING"/>
+            <column header="bic"        name="bic"          type="STRING"/>
+            <column header="url"                            type="skip"/>
+            <column header="adapter_id"                     type="skip"/>
+            <column header="bank_code"  name="bank_code"    type="STRING"/>
+            <column header="idp_url"                        type="skip"/>
+            <column header="aspsp_sca_approaches"           type="skip"/>
+        </loadData>
+        <rollback />
+    </changeSet>
+
+    <changeSet author="iiuex@adorsys.com.ua" id="2020-08-21-2">
```

Comment:
This changeSet should be triggered conditionally, and should not be in `0002-mock-data.xml` it is real banks data, not mock-banks

Reply:
Added context field to the changeSet